quantumarticle.cls

```diff
@@ -294,9 +297,11 @@ class for Quantum - the open journal for quantum science (http://quantum-journal
 	\IfFileExists{lmodern.sty}{
 		\RequirePackage{lmodern}
 	}{
-		\IfFileExists{type1ec.sty}{}{
-			\PackageError{quantumarticle}{Either the lmodern or the cm-super package are required by quantumarticle in order to produce high quality pdf output.}{Please install the lmodern package, or, if this is not possible, the cm-super package.}
-		}
+          \IfFileExists{type1ec.sty}{}{
+            \iftoggle{@noarxiv}{}{%
```

Comment:
Do you think this works? If `noarxiv` is given then the option `a4paper` is executed which sets `@papersizespecified`, which means you will not end up in this branch of the if. 

We should better _set_ the papersize in `noarxiv` if `@papersizespecified` is false (if we don't check this we might turn back an earlier option) but not set `@papersizespecified` to true, shouldn't we?